src/configure.py

```diff
@@ -48,7 +48,7 @@
     'delegatorpaysrafee': "Who is going to pay for 0 balance reactivation/burn fee: 0 for delegator, 1 for delegate. Type enter for delegator",
     'supporters': "Add supporter address. Supporters do not pay bakery fee. Type enter to skip",
     'specials': "Add any addresses with a special fee in form of 'PKH,fee'. Type enter to skip",
-    'final': "Add excluded address in form of 'PKH,target'. Possbile targets are: TOB: leave at balance, TOF: to founders, TOE: to everybody. Type enter to skip"
```

Comment:
why is this removed?

Reply:
Because it is not used. 'excluded addresses' is implemented 4 lines above.